src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -292,18 +290,25 @@ class ResourceService(
 
   /**
     * Overwrites an existing policy's membership (keyed by resourceType/resourceId/policyName) if it exists
-    * @param resourceType
-    * @param policyName
-    * @param resource
+    * @param policyId
     * @param membersList
     * @return
     */
-  def overwritePolicyMembers(resourceType: ResourceType, policyName: AccessPolicyName, resource: FullyQualifiedResourceId, membersList: Set[WorkbenchEmail], samRequestContext: SamRequestContext): IO[Unit] =
+  def overwritePolicyMembers(policyId: FullyQualifiedPolicyId, membersList: Set[WorkbenchEmail], samRequestContext: SamRequestContext): IO[Unit] =
```

Comment:
`overwritePolicyMembers` did not trigger a sync at all 😱 !

Reply:
Oh wow. Yeah, that's not great, good catch.